pagopa/io-app
# feat(Messaggi a valore legale): [IAMVL-21] Read status for MVL

## Short description
Set the message as read when entering the detail screen.

## How to test
1. open the app with MVL active
2. see the new MVL
3. open it
4. go back
5. notice how the read state has changed


https://user-images.githubusercontent.com/2094604/147646112-c9b5326c-2f03-403e-a8fe-c918a8aab36b.mov
